src/langcheck/metrics/custom_text_quality.py

```diff
@@ -11,6 +12,8 @@
 from langcheck.metrics.metric_inputs import get_standard_metric_inputs
 from langcheck.metrics.metric_value import MetricValue
 
+IndividualInputType = Union[str, List[str], None]
```

Comment:
Can't we just import this from langcheck.metrics.metric_inputs?

Reply:
Yeah it should definitely be better, updated